a10_neutron_lbaas/etc/defaults.py

```diff
@@ -37,6 +37,7 @@
 DEVICE_OPTIONAL_DEFAULTS = {
     "protocol": "https",
     "port": 443,
+    "conn-limit": 8000000,
```

Comment:
Discussed this w/ Hunter - as there's no default on the API side and no method of disabling it, leaving the documented system default is the best practice. Perhaps submit a FR to API team to correct this behavior?